rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -1360,6 +1360,20 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 					continue; // system files are not included in the file list
 				}
 
+				if (!g_cfg.misc.allow_filesystem_metadata_in_save_folders && entry.name.starts_with("._"sv))
```

Comment:
Wouldn't this also be a lowercase letter 🧐

Reply:
Lowercase is only a-z. Symbols have no case.